### ISSUE TITLE
Project Setup

### ISSUE THREAD

**Issue by CloudMikhail** [CloudMikhail/Panlingo#1]
This story will be comprised of the following: 

1. Creating a Github repository
2. Setting up a basic folder structure
3. Creating the initial README.md file
4. Setting up a .gitignore file

**Comment by CloudMikhail** [comment]
I have completed setting up my github repo for this project along with creating the basic folder structure. I encountered issues when trying to  push my updates to my repo which I eventually discovered it was due to my git version being out of date. On Monday, I will make a simple readme file which will give a high level overview of what this project is about and how I feel it is impactful on my learning journey. I will also finish making my .gitignore file as well.

**Comment by CloudMikhail** [comment]
- I have created a high level overview readme file that basically explains the use case of this project, my personal reasons on why I wanted to build this application, and how it was created. This readme will be updated throughout the creating of this application. 

- I have also created my .gitignore file that will ensure that unnecessary files aren't tracked by Git. 

** This story can be marked as completed**
